UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -915,18 +918,21 @@ private BoneWeight[] CreateBoneWeightArray(Vector4[] joints, Vector4[] weights,
 
 			return boneWeights;
 		}
-
+
 		/// <summary>
 		/// Ensures each bone weight influences applied to the vertices add up to 1
 		/// </summary>
 		/// <param name="weights">Bone weight array</param>
-		private void NormalizeBoneWeightArray(Vector4[] weights)
+		private void NormalizeBoneWeightArray(Vector4[] weights)
 		{
 			for (int i = 0; i < weights.Length; i++)
 			{
 				var weightSum = (weights[i].x + weights[i].y + weights[i].z + weights[i].w);
-				if (weightSum < 0.98f)
+

```

Comment:
This is the first fix: I have animations where, if weighted to 4 bones, has weight values > 1.

Reply:
Should you use an epsilon value?

Reply:
Yes!